features/yield/FarmingCalculator.tsx

```diff
@@ -65,12 +65,9 @@ const FarmingCalculator = () => {
   );
   const rollToPool = getPoolDataForToken(rollToTokenAddress);
   const rollStartDate = rollFromTokenObj && rollFromTokenObj.rollStartDate;
-  const rollFromTokenName = isRollToToken
-    ? rollFromTokenObj && rollFromTokenObj.rollFromTokenName
-    : rollFromTokenObj && rollFromTokenObj.rollToTokenName;
-  const rollToTokenName = !isRollToToken
-    ? rollFromTokenObj && rollFromTokenObj.rollFromTokenName
-    : rollFromTokenObj && rollFromTokenObj.rollToTokenName;
+  const rollFromTokenName =
```

Comment:
@chrismaree what was the reasoning behind the `isRollToToken` logic? It seemed to be showing the wrong `rollToTokenName` when selecting the `rollFromToken` like the OCT tokens.

Reply:
The logic was to ensure that we always had the right combination of `from` and `to` token names during the roll.

Reply:
This actually worked mid-roll

Reply:
I tested it out pre-roll, post-roll, mid-roll. I think it probably reverses if you don't supply the `rollToToken` heh